app/src/services/transformService.js

```diff
@@ -84,13 +86,15 @@ const transformService = {
 
   confirmationId: ipcPlan => ipcPlan ? ipcPlan.ipcPlanId.split('-')[0].toUpperCase() : undefined,
 
-  ipcResult: (business, contacts, ipcPlan, location) => {
+  ipcResult: (business, contacts, ipcPlan, location, inspectionStatuses) => {
+    const sortedStatuses = inspectionStatuses.sort((a,b) => b.createdAt.getTime() - a.createdAt.getTime());
```

Comment:
As a thought, if we're always going to want to order results based on (reverse) chronological order, it may be prudent to have the dataService do an order by clause instead so that the transform function doesn't need to worry about sorting.

Reply:
yeah, i had an order on that relationship, but it doesn't work.  it doesn't look like that is supported in sequelize. 😞